vineyard/data/ir_ui_view.xml

```diff
@@ -0,0 +1,381 @@
+    <field name="model">stock.quant</field>
+    <field name="name">stock.quant.list.editable</field>
+    <field name="priority">160</field>
+    <field name="active" eval="True"/>
+    <field name="type">list</field>
+  </record>
+  <record id="stock_picking_form_view_inherit" model="ir.ui.view">
+    <field name="arch" type="xml">
+      <data>
+        <xpath expr="//form[1]/sheet[1]/group[1]/group[1]/field[@name='picking_type_id']" position="after">
+          <field name="location_id" hidden="not x_is_harvest" domain="[(&quot;x_is_harvest&quot;, &quot;!=&quot;, False)]"/>
+        </xpath>
+      </data>
+    </field>
+    <field name="inherit_id" ref="stock.view_picking_form"/>
+    <field name="mode">extension</field>
+    <field name="model">stock.picking</field>
+    <field name="name">stock.picking.form</field>
+    <field name="priority">160</field>
+    <field name="active" eval="True"/>
+    <field name="type">form</field>
+  </record>
+  <record id="stock_vpicktree_inherit" model="ir.ui.view">
+    <field name="arch" type="xml">
+      <data>
+        <xpath expr="//field[@name='origin']" position="replace"/>
+        <xpath expr="//field[@name='partner_id']" position="replace"/>
+        <xpath expr="//field[@name='location_dest_id']" position="replace"/>
+        <xpath expr="//list[1]" position="attributes">
+          <attribute name="default_order">date_done desc</attribute>
+        </xpath>
+        <xpath expr="//list[1]/field[@name='name']" position="after">
+          <xpath expr="//field[@name='scheduled_date']" position="move"/>
+          <xpath expr="//field[@name='date_done']" position="move"/>
+        </xpath>
+        <xpath expr="//field[@name='user_id']" position="after">
+          <xpath expr="//field[@name='state']" position="move"/>
+        </xpath>
+        <xpath expr="//field[@name='date_done']" position="attributes">
+          <attribute name="optional">show</attribute>
+        </xpath>
+      </data>
+    </field>
+    <field name="inherit_id" ref="stock.vpicktree"/>
+    <field name="mode">extension</field>
+    <field name="model">stock.picking</field>
+    <field name="name">stock.picking.list</field>
+    <field name="priority">160</field>
+    <field name="active" eval="True"/>
+    <field name="type">list</field>
+  </record>
+  <record id="stock_move_graph_view_inherit" model="ir.ui.view">
+    <field name="arch" type="xml">
+      <data>
+        <xpath expr="//field[@name='product_id']" position="replace">
+          <field name="date" type="col"/>
+        </xpath>
+        <xpath expr="//field[@name='location_dest_id'][not(ancestor::field)]" position="replace">
+          <field name="location_id" type="col"/>
+        </xpath>
+        <xpath expr="//field[@name='product_uom_qty']" position="replace">
+          <field name="product_qty" type="measure"/>
+        </xpath>
+      </data>
+    </field>
+    <field name="inherit_id" ref="stock.view_move_graph"/>
```

Comment:
Since you replace everything, better create a new view (and make sure this one is used instead)